modules/ldap/lib/Auth/Process/BaseFilter.php

```diff
@@ -142,6 +142,7 @@ public function __construct(&$config, $reserved)
             $authconfig['ldap.port']       = @$authsource['port'];
             $authconfig['ldap.timeout']    = @$authsource['timeout'];
             $authconfig['ldap.debug']      = @$authsource['debug'];
+            $authconfig['ldap.referrals']   = (@$authsource['referrals'] ? @$authsource['referrals'] : false);
```

Comment:
I think this should default to `true` since `SimpleSAML_Auth_LDAP::__construct` does.